web/scripts/start-console.sh

```diff
@@ -37,8 +37,24 @@ fi
 CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
 CONSOLE_PORT=${CONSOLE_PORT:=9000}
 
+function checkClusterEndpoint() {
+    echo "Checking cluster endpoints..."         
+    previous="$(grep 'BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=' $(pwd)/scripts/env.list | cut -d= -f2-)"
```

Comment:
I think you can drop the `pwd` and get the same result.

```suggestion
    previous="$(grep 'BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=' scripts/env.list | cut -d= -f2-)"
```